src/tpm2_param_enc.c

```diff
@@ -0,0 +1,282 @@
+/* tpm2_param_enc.c
+ *
+ * Copyright (C) 2006-2020 wolfSSL Inc.
+ *
+ * This file is part of wolfTPM.
+ *
+ * wolfTPM is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * wolfTPM is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1335, USA
+ */
+
+
+#include <wolftpm/tpm2_param_enc.h>
+#include <wolftpm/tpm2_packet.h>
+
+#include <wolfssl/wolfcrypt/hmac.h>
```

Comment:
This file needs to be wrapped in `#ifndef WOLFTPM2_NO_WOLFCRYPT`

Reply:
Indeed. Thank you